src/contacts/requirements.txt

```diff
@@ -1,107 +1,101 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile --output-file=requirements.txt requirements.in
+#    pip-compile --output-file=/Users/obourgeois/workspace/repos/bank-of-anthos/src/contacts/requirements.txt /Users/obourgeois/workspace/repos/bank-of-anthos/src/contacts/requirements.in
```

Comment:
Should we redact file paths from local environment?

Reply:
Oops! Fixed :)